src/datetime/serde.rs

```diff
@@ -23,10 +23,9 @@ pub struct MicroSecondsTimestampVisitor;
 #[derive(Debug)]
 pub struct MilliSecondsTimestampVisitor;
 
-/// Serialize into an ISO 8601 formatted string.
+/// Serialize into an ISO 8601 formatted string, with a format similar to RFC 3339.
```

Comment:
This seems weirdly imprecise, and its precision doesn't match the string used for `expecting()`.

Reply:
It is the RFC 3339 format with the addition that it can (de)serialize datetimes outside the `0..=9999` years range using the ISO 8601 syntax. I'll try to word it better.